index.js

```diff
@@ -0,0 +1,141 @@
+module.exports = {
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    ecmaVersion: 2018,
+    ecmaFeatures: {
+      jsx: true,
+    },
+    sourceType: 'module',
+    project: './tsconfig.json',
+  },
+  extends: ['plugin:react/recommended', 'plugin:prettier/recommended'],
```

Comment:
Is there a reason we're not extending airbnb (like we have done in the past)?

Reply:
No, just forgot it. I will add it